test/org/sagebionetworks/bridge/models/schedules/ActivitySchedulerTest.java

```diff
@@ -53,7 +53,7 @@ public void before() {
         plan.setGuid("BBB");
         
         // Day of tests is 2015-04-06T10:10:10.000-07:00 for purpose of calculating expiration
-        DateTimeUtils.setCurrentMillisFixed(1428340210000L);
+        DateTimeUtils.setCurrentMillisFixed(NOW.getMillis());
```

Comment:
These were different, which is really confusing. Now freezing system time to NOW which is what you'd expect, this did change the dates in the test but it's all consistent with expectations